keyboards/fc660c/keymaps/spacebarracecar/config.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include "config_common.h"
```

Comment:
This isn't actually needed, as the `config_common.h` file should be included in the keyboard's config.h.